azurelinuxagent/ga/update.py

```diff
@@ -132,7 +140,11 @@ def __init__(self):
         self.protocol_util = get_protocol_util()
 
         self._is_running = True
+
+        # Member variables to keep track of the Agent AutoUpgrade
         self.last_attempt_time = None
+        self._last_hotfix_upgrade_time = None
```

Comment:
could we initialize these 3 members to something other than None? maybe a time in the past, if that make sense

Reply:
The legacy code had a None check so I just kept it as is. I can modify that part, shouldn't be a big change

Reply:
Yes, I noticed the legacy code initializes to None, and that is not a good practice in general. I'm ok leaving it as is, but let's try not to follow those patterns in new code (unless the cost is excessive)

Reply:
Makes sense, I'll update this if I touch this code for GAVersioning